app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_user_posts_counter
```

Comment:
- [ ] This will only be triggered after creation, but it should after save so I will encourage you to replace it with after_save.